src/autoscaler/pruner/cmd/pruner/pruner_suite_test.go

```diff
@@ -66,16 +66,17 @@ func initConsul() {
 
 func initConfig() {
 	cfg.Logging.Level = "debug"
+	dbUrl := os.Getenv("DBURL")
```

Comment:
It was moved as a part of refactoring as the same code was repeated. The sqldb_suite_test, covers the $DBURL not set case. Am I missing something. As os.Getenv("DBURL") is being used all over the place.